### PR TITLE
Adding Panasonic Viera remote documentation

### PR DESCRIPTION
**Pull request by joogps**
## Proposed change

This pull request adds some information about the new `remote` platform being added to the Panasonic Viera integration documentation page. I was also thinking of writing a section about how to deal with the "Couldn't gather device info" error ([check this answer for details](https://github.com/home-assistant/core/issues/42205#issuecomment-715345337)), but I'm not really sure if that's really essential.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/42450
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by joogps** [comment]
uh...

**Comment by homeassistant** [comment]
Hello @joogps,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`92d6de48b32f7be7f60664c669b2e0bb8e8fb290`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant.io&#x2F;pull&#x2F;15377&#x2F;commits&#x2F;92d6de48b32f7be7f60664c669b2e0bb8e8fb290) - This commit has something that looks like an email address (joaogabrielps13@hotmail.com). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

File: source/_integrations/panasonic_viera.markdown [hunk 1]
```diff
@@ -13,9 +14,20 @@ The `panasonic_viera` platform allows you to control a Panasonic Viera TV.
...
+There is currently support for the following device types within Home Assistant:
+
+- Media Player
+- [Remote](#remote)
+
+## Configuration
+
+To configure your Panasonic Viera TV, head to the **Configuration > Integrations** page. Click on the plus (+) button to add a new integration.
+
+Once the integration is loaded, with your TV turned on and connected to your local network, enter the IP address of your TV and a name of your choice.
```
Comment (frenck) [inline source/_integrations/panasonic_viera.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

File: source/_integrations/panasonic_viera.markdown [hunk 2]
```diff
@@ -88,6 +100,23 @@ script:
...
+- up
+- down
+- left
+- right
+- select
+- home
+- back
+- power
+
+The list with all known valid keys can be found [here](https://github.com/florianholzapfel/panasonic-viera/blob/521cefadc8e1543514ce41d3d49e9218d1c2302d/panasonic_viera/__init__.py#L35). Additionally, you can also send custom commands, such as `"NRC_HOME-ONOFF"` (which is the same as `home`).
```
Comment (frenck) [inline source/_integrations/panasonic_viera.markdown]:
```suggestion
When the integration is configured, two entities will be created: a `media_player` and a `remote`. The remote allows you to send key commands to your TV with the `remote.send_command` service.

Some of the known valid key values are:

- `up`
- `down`
- `left`
- `right`
- `select`
- `home`
- `back`
- `power`

The list with all known valid keys can be found [here](https://github.com/florianholzapfel/panasonic-viera/blob/521cefadc8e1543514ce41d3d49e9218d1c2302d/panasonic_viera/__init__.py#L35). Additionally, you can also send custom commands, such as `"NRC_HOME-ONOFF"` (which is the same as `home`).
```

**Review by frenck — APPROVED** [review]
Thanks, @joogps 👍

**Pull request merged**